pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1690,9 +1690,7 @@ class HdRprApiImpl {
 
             if (m_state == kStateRender && config->IsDirty(HdRprConfig::DirtyRenderQuality)) {
                 TfToken activeRenderQuality;
```

Comment:
After remove "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable